src/main/java/zas/admin/zec/backend/conversations/Question.java

```diff
@@ -5,16 +5,23 @@
 import java.util.List;
 
 public record Question(
```

Comment:
Keep the @Nullable annotation on not mandatory fields
Boolean can be boolean (defaulting to false)
Integer can be int (defaulting to 0)